examples/pytorch/preprocessing_dist_training/substitute_to_hetero.py

```diff
@@ -0,0 +1,24 @@
+import argparse
+import os
+import glob
+import pandas as pd
```

Comment:
this script tries to fix the bug in ParMETIS?

Reply:
yes exactly This try to fix bug in Parmetis